### ISSUE TITLE
Leave control of is_superuser status with Django by default

### ISSUE THREAD

**Issue by Andras1000** [AzMoo/django-okta-auth#14]
In `tokens.py` the `is_superuser` status of a user will always be overwritten by this library.

```
user.is_superuser = bool(
    self.config.superuser_group
    and "groups" in claims
    and self.config.superuser_group in claims["groups"]
)
```

Make this code optional via settings so when an existing user is authenticated the already existing `is_superuser` status can be used. 

I ran into this limitation as the `groups` scope is not returned in the flow unless one has the API Access Management feature in Okta which allows custom authorization server (see [here](https://support.okta.com/help/s/article/Okta-Groups-or-Attribute-Missing-from-Id-Token)), so could not use the `SUPERUSER_GROUP` setting.

Same applies to `is_staff`.

**Comment by samkuehn** [comment]
I am having the same issue (also changes user.is_staff).  Here are my comments/suggestions on another issue.

https://github.com/AzMoo/django-okta-auth/issues/8#issuecomment-809679256

**Comment by AzMoo** [comment]
Resolved in #28
